clients/web/src/main.js

```diff
@@ -1,3 +1,30 @@
+import $ from 'jquery';
+import _ from 'underscore';
+import Backbone from 'backbone';
+import moment from 'moment';
 import * as girder from 'girder';
 
 window.girder = girder;
+
+// Some cross-browser globals
+if (!window.console) {
+    window.console = {
+        log: $.noop,
+        error: $.noop
+    };
+}
+
+// For testing and convenience, available now because of testUtils.js reliance on $
+window.$ = $;
+window._ = _;
+window.moment = moment;
+window.Backbone = Backbone;
+
+if (!window.disableGirderMainApp) {
```

Comment:
Library bundles aren't able to execute code at import time (i.e. they cannot have an entry point). So it seemed like a good idea to put this inside the main bundle, in the interest of having all of our javascript code in files rather than in the HTML directly.

What's the reason to prefer it in the mako template?